IotHubConsumer/Program.cs

```diff
@@ -9,13 +10,28 @@ class Program
         static void Main(string[] args)
         {
             var dir = System.Reflection.Assembly.GetExecutingAssembly().Location;
-
             var configuration = Configuration.BuildConfiguration();
 
-            Console.WriteLine("Enter message payload for C2D");
-            var input = Console.ReadLine();
-            var consumer = new Consumer(configuration);
-            consumer.SendCloudToDeviceMessageAsync(input).Wait();
+            //Console.WriteLine("Enter message payload for C2D");
+            // var input = Console.ReadLine();
+            var consumer = new SenderConsumer(configuration);
```

Comment:
You've changed this function in a way that no messages are sent from here anymore. Perhaps we should move the Sender consumer into it's own project with it's own program, since we can't run both of them from here anymore? And then not call it a sender consumer anymore :)

Reply:
Good point!  Will think how I can change it.